src/figuresClasses.ts

```diff
@@ -1,19 +1,95 @@
 export interface Figure {
+  shape: string;
+  color: string;
 
+  getArea(): number;
 }
 
-export class Triangle {
+function round(expression: number): number {
+  return Math.floor(expression * 100) / 100;
+}
+
+enum ShapeFigure {
+  triangle = 'triangle',
+  circle = 'circle',
+  rectangle = 'rectangle'
+}
 
+enum ColorFigure {
+  red = 'red',
+  green = 'green',
+  blue = 'blue'
 }
 
-export class Circle {
+export class Triangle implements Figure {
+  constructor(
+    public color: string = ColorFigure.green,
+    public a: number,
+    public b: number,
+    public c: number,
+    public shape: string = ShapeFigure.triangle,
```

Comment:
```suggestion
    public shape: ShapeFigure,
```
and it shouldn't be in the constructor. We don't want to declare it when creating the new instance, for example:
```
new Triangle(...someOtherProps, ShapeFigure.Triangle)
```

we already creating `Triangle`